keyboards/handwired/jotlily60/info.json

```
@@ -0,0 +1,111 @@
+{
+    "manufacturer": "Jotix",
+    "keyboard_name": "jotlily60",
+    "maintainer" : "jotix",
+    "url": "https://github.com/qmk/qmk_firmware/tree/master/keyboards/handwired/jotlily60",
+    "usb": {
+        "vid": "0x4A4F",
+        "pid": "0x2332",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true
```

It's generally preferred that mouse key functionality is enabled. Command and console generally aren't needed by default.
```suggestion
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": true,
        "nkro": true
```